.markdown-link-check-config.json

```diff
@@ -0,0 +1,43 @@
+{
+  "aliveStatusCodes": [429, 200],
+  "ignorePatterns": [
+    {
+      "pattern": "^http://www5.in.tum.de/pub/int/"
+    },
+    {
+      "pattern": "^https://onlinelibrary.wiley.com/"
+    },
+    {
+      "pattern": "^http://dx.doi.org/"
+    },
+    {
+      "pattern": "^https://www.researchgate.net/"
+    },
+    {
+      "pattern": "^https://pkgs.org/"
+    },
+    {
+      "pattern": "^https://packages.ubuntu.com/"
+    },
+    {
+      "pattern": "^https://launchpad.net/"
+    },
+    {
+      "pattern": "^https://www.comsol.com/"
+    }
+  ],
+  "replacementPatterns": [
+    {
+      "pattern": "^/",
+      "replacement": "https://precice.org/"
```

Comment:
This will break local builds if run on the source.

Reply:
Thanks! Replaced it with `"replacement": "{{BASEURL}}/"`, which should work locally as well: https://github.com/tcort/markdown-link-check/pull/145

Reply:
But this should also not really help anywhere at the moment (but I cannot add a comment, as this is a json file).